docs/ux.md

```diff
@@ -0,0 +1,174 @@
+
+## Content
+
+- [Principles](#principles)
+- [Connecting physical resources to slurm resoures and how to use them](#connecting-physical-resources-to-slurm-resoures-and-how-to-use-them)
+  - [HPC admin scope](#hpc-admin-scope)
+  - [HPC user scope](#hpc-user-scope)
+  - [HPC application scope](#hpc-application-scope)
+  - [Backend specifics](#backend-specifics)
+    - [IBM Direct Access API](#ibm-direct-access-api)
+    - [Qiskit Runtime Service](#qiskit-runtime-service)
+- [Examples](#examples)
+  - [Running jobs with dependencies](#running-jobs-with-dependencies)
+  - [Running a job with several slurm QPU resources](#running-a-job-with-several-slurm-qpu-resources)
+  - [Running primitives directly](#running-primitives-directly)
+  - [Other workflow tools](#other-workflow-tools)
+
+See [Overview](./overview.md) for a glossary of terms.
+
+## Principles
+
+Slurm QPU resource definitions determine what physical resources can be used by slurm jobs.
+User source code should be agnostic to specific backend instances and even backend types as far as possible.
+This keeps source code portable while the QPU seletion criteria are part of the resource definition (which is considered configuration as opposed to source code).
+The source code does not have to take care resp. is not involved in resource reservation handling (that is done when slurm jobs are assigned QPU resources and start running, if applicable on the backend) or execution modes like sessions (these are automatically in place while the job is running, if applicable on the backend).
+This makes the source code more portable between similar QPU resource types through different backend access methods (such as IBM's Direct Access API and IBM's Qiskit Runtime service through IBM Quantum Platform).
+All backend types (such as IBM's Direct Access API, IBM's Qiskit Runtime service, or Pasqal's backends) follow these principles.
+
+## Connecting physical resources to slurm resoures and how to use them
+
+Note the exact syntax is subject to change -- this is a sketch of the UX at this time.
+
+### HPC admin scope
+
+HPC administrators configure, what physical resources can be provided to slurm jobs.
+Note sensitivethe config could contain sensitive information and m
+
+```
+# slurm quauntum plugin configuration
+
+# DA backend
+name=da-local-backend                                                    \
+url=https://da-endpoint.my-local.domain/                                 \
+da_crn=crn:v1:bluemix:public:quantum-computing:us-east:a/43aac17...      \
+da_apikey_file=/root/da_apikey                                           \
+s3_url=https://s3.my-local.domain/...                                    \
+s3_accesstoken_file=/root/s3_accesstoken
+
+# QRS backends
+name=ibm_fez                                                             \
+type=qiskit-runtime-service-ibmcloud
+
+name=ibm_marrakesh                                                       \
+type=qiskit-runtime-service-ibmcloud
```

Comment:
I might just not have seen this config format before, but where does this get parsed and how will that system handle repeating keys (e.g. the `name` key being re-used without "sections")?

Reply:
the format aligns to what seems typical of gres.conf content. Adding a comment.